jax/experimental/jax2tf/examples/tf_js/quickdraw/README.md

```diff
@@ -0,0 +1,8 @@
+# JAX2TF.js Quickdraw example
+
+This example contains the code necessary to train a CNN on the Quickdraw
+dataset using Flax, and subsequently convert it using JAX2TF into a model that
+can be loaded in TF.js.
+
```

Comment:
Can you please add some text about what the user is supposed to run, and what to look for? Also perhaps what to do with the tarball and how it fits with the rest of the code here?

Reply:
PTAL at the latest README and let me know what you think!